src/database/mod.rs

```diff
@@ -297,4 +301,17 @@ impl ConnectOptions {
         self.test_before_acquire = value;
         self
     }
+
+    /// Set the application name for the connection (PostgreSQL only)
+    #[cfg(feature = "sqlx-postgres")]
```

Comment:
I think we can remove all feature guards. The value will be ignored by other backends anyway.